### DISCUSSION TITLE
[COST] How to estimate the cost of API(get/put) for S3 or other OSS.

### DISCUSSION THREAD

**Discussion by ximply** [openobserve/openobserve#1224] [General]
Storage is just a part of cost for S3 or other OSS.

**Comment by prabhatsharma** [comment]
There are a lot of changes happening in how we store and query s3. Any answer we provide today may not be true very soon. We should be able to provide some guidance around this in 1 month.

**Comment by davidykh** [comment]
Sorry late joiner to the party, currently setup my playground on OCI and want to do logging and monitoring.

I have the same question as OP, since OCI pricing consist of both storage and API call, free tier offers 50,000 S3/object storage API call per month, I would like to know how to calculate/estimate the amount of API usage of OO.
Does it write for every single log or entries received, or it cached for a certain time/amount and write in batch? Is there any way to adjust?

Thank you.
